aviatrix_controller_arm/accept_license.py

```diff
@@ -1,18 +1,34 @@
-import subprocess
 import json
+import subprocess
 
 # Get the details if Azure Marketplace image terms
-process = subprocess.Popen(['az','vm', 'image',
-                            'terms', 'show', '--urn',
-                            'aviatrix-systems:aviatrix-bundle-payg:aviatrix-enterprise-bundle-byol:latest'],
-                            stdout=subprocess.PIPE)
+process = subprocess.Popen(
+    [
+        "az",
+        "vm",
+        "image",
+        "terms",
+        "show",
+        "--urn",
+        "aviatrix-systems:aviatrix-bundle-payg:aviatrix-enterprise-bundle-byol:latest",
+    ],
+    stdout=subprocess.PIPE,
+)
 
 out = process.communicate()[0]
```

Comment:
Would be better to put this inside functions rather than while the module is being imported